### DISCUSSION TITLE
Biotite 1.0.2 or 1.1.0

### DISCUSSION THREAD

**Discussion by Croydon-Brixton** [biotite-dev/biotite#721] [General]
Hi @padix-key , 
I believe there was a major update to the CCD end of September - [see e.g. here](https://www.rcsb.org/ligand/CCD), looks like the first one in decades.

I think it'd be good to release biotite v1.0.2/v1.1.0 as soon as we can such that people will have this updated version of the CCD shipped with biotite. The update would probably warrant a `v1.1.0` rather than a patch increment

**Comment by padix-key** [comment]
Actually I planned to release Biotite 1.1.0 already 2-3 weeks ago. However currently there are some problems with DSSP and SRA-Tools in the environment that builds the docs (will be fixed in #720) and there is a change in how the internal CCD can be installed in #716, which I would also like to include in the next release. Both of these are rather small changes, so we are almost there 😃. So you can expect Biotite 1.1.0 before the end of the year.

**Reply by Croydon-Brixton** [reply]
That sounds great to me, thank you for the update!
